server/repository/src/migrations/v2_04_00/add_cold_storage_type_table.rs

```diff
@@ -0,0 +1,34 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "add_cold_storage_type_table"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                CREATE TABLE IF NOT EXISTS cold_storage_type (
+                    id TEXT NOT NULL PRIMARY KEY,
+                    name TEXT NOT NULL,
+                    min_temperature REAL,
+                    max_temperature REAL
+                );
+            "#
+        )?;
+
+        sql!(
+            connection,
+            r#"
+                ALTER TABLE location
+                ADD COLUMN IF NOT EXISTS cold_storage_type_id TEXT
```

Comment:
```suggestion
                ADD COLUMN cold_storage_type_id TEXT
```

IF NOT EXISTS doesn't work with SQLite sadly :(